tools/osbuild-image-info

```diff
@@ -2567,6 +2571,58 @@ def partition_is_lvm(part: Dict) -> bool:
     return part["type"].upper() in ["E6D6D379-F507-44C2-A23C-238F2A3DF928", "8E"]
 
 
+def parse_subvol_list(output):
+    """
+    Parse the output of 'btrfs subvolume list' and return just the subvolume names/paths.
+    """
+    paths = []
+    for line in output.strip().split("\n"):
+        # subvolume names can have spaces in them, but they are the last field and they are preceded by the word
+        # path
+        parts = line.partition(" path ")
+        if len(parts) != 3:
```

Comment:
Maybe better to check for `parts[2] == ""` here for the invalid case, from the docs: `If the separator is not found, returns a 3-tuple containing the original string`

Reply:
-_- sorry, sloppy